README.md

```diff
@@ -21,6 +21,10 @@ BSF Changelog plugin is used for building multi products changelog website withi
 
 ## Changelog ##
 
+### Version 1.0.5 ###
+* New: Added anchor support to changelog headings.
+* Fix: Multiple Products checkbox issue for multiple products changelog.
```

Comment:
Can you describe what the issue was?
General rule of thumb - avoid the word "issue" and describe it bit more